src/driver.ml

```diff
@@ -900,6 +939,34 @@ struct
   let set x = t.data <- Some x
 end
 
+(*$*)
+
+let error_to_str_extension error =
+  let loc = Location.none in
```

Comment:
Yes, it's because it is the location of the error extension node, not the location of the error itself.
The extension node has been added to the AST after parsing, it has no location in the `.ml` or `.mli` file.

Reply:
Yes, it's because it is the location of the error extension node, not the location of the error.
The extension node has been added to the AST after parsing, it has no location in the `.ml` or `.mli` file.

Reply:
> The extension node has been added to the AST after parsing, it has no location in the .ml or .mli file.

Notice that any node added by a ppx rewriter or ppxlib is added to the AST after parsing and still it's important to give those nodes a good location. More concretely, nodes added by the parser have a natural location in the source code file, whereas the locations of nodes added by a ppx rewriter / ppxlib need to be clarified by the author of the rewriter.

Anyways, I've just seen that we were already using `Location.none` in this context before this PR and you've only moved it, so let's keep it like that for now while I'll find out what would be the right thing to do in this context for the future.